commons/commons-test/src/main/java/com/clicktravel/common/test/BeanAssert.java

```diff
@@ -106,6 +107,7 @@ private static Object getPropertyValue(final Class<?> propertyClass) {
         supportedPropertyTypes.put(Character.class, randomChar());
         supportedPropertyTypes.put(String.class, randomString(1000));
         supportedPropertyTypes.put(Date.class, new Date(randomDateTime().getMillis()));
+        supportedPropertyTypes.put(DateTime.class, randomDateTime());
```

Comment:
i belive travel use local objects to track things such as flights and check in times as they are not subject to timezones